PWGJE/EMCALJetTasks/AliEmcalJetTask.cxx

```diff
@@ -1105,7 +1105,17 @@ void AliEmcalJetTask::SetArtificialTrackingEfficiencyFromYAML() {
   auto userInfo = fInputHandler->GetUserInfo();
   AliProdInfo prodInfo(userInfo);
   std::string period = prodInfo.GetAnchorProduction().Data();
-  AliInfoStream() << "anchor production = " << prodInfo.GetAnchorProduction()<< "\n";
+  if(period.size() == 0) {
+    // MC where anchor production is not saved - get the name of the MC production instead
+    period = prodInfo.GetTag(AliProdInfo::kProdTag).Data();
+    if(period.size() == 0) {
+      AliFatal("No information relating to anchored datset or MC tag in this MC production - can't get pT-dependent tracking efficiencies");
+    }
+    AliInfoStream() << "Get MC set tracking efficiency for " << prodInfo.GetTag(AliProdInfo::kProdTag)<< "\n";
```

Comment:
A bit pedantic, but I think it would be better to reduce the maintenance burden by using:

```suggestion
    AliInfoStream() << "Get MC set tracking efficiency for " << period << "\n";
```

Reply:
done